examples/tutorials/core_api_mnist/distributed.yaml

```diff
@@ -0,0 +1,37 @@
+name: coreapi_mnist_tutorial_distributed
+entrypoint: >-
+   python -m determined.launch.torch_distributed
+   python3 model_def_distributed.py
```

Comment:
prolly be consistent about `python` vs `python3`.  If they both work in the image, then it doesn't matter which you pick.